clients/ts/signalr/src/IHttpConnectionOptions.ts

```diff
@@ -38,4 +39,16 @@ export interface IHttpConnectionOptions {
      * Negotiation can only be skipped when the {@link transport} property is set to 'HttpTransportType.WebSockets'.
      */
     skipNegotiation?: boolean;
+
+    /** A constructor that can be used to create a WebSocket.
```

Comment:
Put a comment above these indicating that these are for unit testing or super intrepid code spelunkers. We'll be providing automatic polyfills in Node JS